src/server/conn.rs

```diff
@@ -394,6 +394,22 @@ impl<E> Http<E> {
         self
     }
 
+    /// Configures the maximum number of pending reset streams allowed before a GOAWAY will be sent.
+    ///
+    /// This will default to whatever the default in h2 is. As of v0.3.17, it is 20.
```

Comment:
nit: is there documentation in `h2` we can link to for this?

Reply:
also, silly docs nit: I might say 
```suggestion
    /// This will default to the default value set by the [`h2` crate](https://crates.io/crates/h2).
    /// As of v0.3.17, it is 20.
```
or something, to make it obvious that when we say `h2` here, we are referring to the library?